mne/decoding/search_light.py

```diff
@@ -76,13 +77,22 @@ def fit(self, X, y, **fit_params):
             _sl_fit, self.n_jobs, max_jobs=X.shape[-1], verbose=False)
         self.estimators_ = list()
         self.fit_params = fit_params
+
         # For fitting, the parallelization is across estimators.
-        mesg = 'Fitting %s' % (self.__class__.__name__,)
-        with ProgressBar(X.shape[-1], mesg=mesg, position=self.position) as pb:
+        if self.verbose:
```

Comment:
Rather than this pattern that results in code dup of the loop, I generally use:
```
if self.verbose:
    ctx = ProgressBar(...)
else:
    ctx = contextlib.nullcontext()
with ctx:
    ...
```